frontend/src/pages/EditReview.tsx

```diff
@@ -27,6 +28,7 @@ export function ReviewView({
   };
 
   useEffect(() => {
+    document.title = "TSE Fulcrum - Application Review";
```

Comment:
The page title is set to the applicant's name in ApplicationView (though I think I merged that after you started this PR); can you check if this conflicts? Feel free to change that part too though.